rhpam-config/7.13.0/common.yaml

```diff
@@ -561,10 +561,24 @@ smartRouter:
                   - name: KIE_SERVER_ROUTER_TLS_KEYSTORE
                     value: "/etc/smartrouter-secret-volume/keystore.jks"
                   #[[end]]
+                  #[[if not .SecretAdminCredentials]]
                   - name: KIE_ADMIN_USER
                     value: "[[.AdminUser]]"
                   - name: KIE_ADMIN_PWD
                     value: "[[.AdminPassword]]"
+                  #[[end]]
+                  #[[if .SecretAdminCredentials]]
+                  - name: KIE_ADMIN_USER
+                    valueFrom:
+                      secretKeyRef:
+                        name: "[[.SecretAdminCredentials]]"
+                        key: username
+                  - name: KIE_ADMIN_PWD
+                    valueFrom:
+                      secretKeyRef:
+                        name: "[[.SecretAdminCredentials]]"
+                        key: password
+                  #[[end]]
```

Comment:
Here is user from secret applied only for Smart Router deployment. Username and password from secret should be provided for all deployments.

Reply:
wrong copy of the old PR